mis_builder_budget_contrib/readme/HISTORY.rst

```diff
@@ -0,0 +1,6 @@
+15.0.1.0.0 (2022-11-24)
```

Comment:
if you don't want to maintain this file you can use newsfragments https://github.com/OCA/maintainer-tools/tree/master/template/module#automatic-changelog-generation

Reply:
https://github.com/OCA/mis-builder/commit/f861cd22faded606bcbff4a67d7bf8b5fff72ee4#diff-57d164ab6c42a284162b95ab7af7b65a582ebe65e3efcc79e6aa04a70db2a995
This way ?

Reply:
hmm, no, you don't need all that work. OCA quality tools will take care of rendering, you just have to put file for your change in `newsfragments` folder.
NOTE: the changelog is not mandatory, you can skip it.